acceptance/tests/mesh_v2/mesh_inject_test.go

```diff
@@ -74,9 +74,11 @@ func TestMeshInject_MultiportService(t *testing.T) {
 			require.Len(t, podList.Items, 1)
 			require.Len(t, podList.Items[0].Spec.Containers, 3)
 
-			{
-				// TODO: once ACLs are implemented, only run this block when secure=true and delete the below line and fixture since the default is deny when secure is true.
+			if !secure {
 				k8s.KubectlApplyK(t, ctx.KubectlOptions(t), "../../tests/fixtures/cases/trafficpermissions-deny")
+			}
+
+			{
```

Comment:
Is this set of parens required on L81 and L94? Also non-blocking, I don't think it's worth waiting for another pipeline run to change this

Reply:
Not required. I thought maybe it was scoping this step, but I will remove it.